mlflow/models/model.py

```diff
@@ -26,6 +26,9 @@
     "have been logged successfully under %s. Set logging level to DEBUG via "
     '`logging.getLogger("mlflow").setLevel(logging.DEBUG)` to see the full traceback.'
 )
+_LOG_MODEL_MISSING_SIGNATURE_WARNING = (
+    "Missing model signature. Please add signature with `mlflow.models.add_signature`."
```

Comment:
Do we have the api `mlflow.models.add_signature` ?

Reply:
We do now 😉 See https://github.com/mlflow/mlflow/pull/8490